package.json

```diff
@@ -119,9 +120,11 @@
     "lodash": "4.17.11",
     "nodelist-foreach-polyfill": "^1.2.0",
     "prop-types": "^15.6.0",
+    "react-async-script": "^1.0.1",
     "react-dom-factories": "^1.0.2",
     "react-intl": "^2.4.0",
     "react-measure": "2.3.0",
+    "react-transition-group": "2.2.1",
```

Comment:
Why was this added? We don't seem to use it anywhere.

Reply:
I think it was needed peer dependency from something i will check that again. Thanks for catch

Reply:
It's peer dependency for `recharts` which is used as example of custom visualization